src/main/scala/guessbook/GuessBookUserInput.scala

```diff
@@ -0,0 +1,206 @@
+package guessbook
+
+import guesswho.GenderEnum
+
+import scala.io.StdIn
+import scala.util.{Failure, Success, Try}
+
+object GuessBookUserInput {
+  def main(args: Array[String]): Unit = {
+    val game = GuessBook
+    println("START OF GAME\n")
+
+    // Randomly select a book for each player
+//    val selectedBooks = List(game.selectBook(game.allBooks), game.selectBook(game.allBooks))
+
+    // Each player selects the book that the other one must guess
+    val selectedBookP2: Book = selectBookForOpponent(1)
+    val selectedBookP1: Book = selectBookForOpponent(2)
+    val selectedBooks = List(selectedBookP1, selectedBookP2)
+
+    val booksOnBoard = Array(game.allBooks, game.allBooks)
+    var gameOver = false
+    var guesser = 2
+
+    while (!gameOver) {
+      guesser = guesser % 2 + 1 // 1 -> 2, 2 -> 1
+      println(s"\nPlayer $guesser's turn!")
+      println(s"Books remaining on P$guesser's board:")
+      game.printRemainingBooks(booksOnBoard(guesser-1), maxBooksPerLine = 5)
+
+      val guessType: Int = chooseAttributeOrTitle()
+      guessType match {
+        case 1 => { // guessing attribute and value
+          val guessAttribute: BookAttribute = specifyGuessAttribute()
+          guessAttribute match {
+            case Honkaku | SchoolOrUniversityStudents | Series | PushkinVertigo => {
+              val guessValue: Boolean = specifyBooleanValue()
+              booksOnBoard(guesser-1) = game.poseQuestion(selectedBooks(guesser-1), guessAttribute, guessValue, booksOnBoard(guesser-1))
+            }
+            case AuthorGender => {
+              val guessValue: GenderEnum.Value = specifyEnumValue(GenderEnum)
+              booksOnBoard(guesser-1) = game.poseQuestion(selectedBooks(guesser-1), guessAttribute, guessValue, booksOnBoard(guesser-1))
+            }
+            case CountryOfOrigin => {
+              val guessValue: Country.Value = specifyEnumValue(Country)
+              booksOnBoard(guesser-1) = game.poseQuestion(selectedBooks(guesser-1), guessAttribute, guessValue, booksOnBoard(guesser-1))
+            }
+            case DetectiveType => {
```

Comment:
maybe `guesser-1` can be extracted somewhere so that it can be represented as like _otherPlayer_?